views/templates.pug

```diff
@@ -116,6 +116,12 @@
           button.dropdown-item.btn.btn-light.btn-sm.view-cfg(title="Show Graph Output")
             span.fas.fa-exchange-alt
             | &nbsp;Graph output
+          button.dropdown-item.btn.btn-light.btn-sm.view-clangtidytool(title="Clang-tidy")
```

Comment:
Idea: "Tools" submenu where then we list only the tools, as not to clutter this one

Reply:
Not a bad idea, not sure what that would feel like, but we can try

Reply:
Bootstrap removed the submenus on v3. But there's a better idea